op-node/node/api.go

```diff
@@ -71,6 +74,22 @@ func (n *adminAPI) SequencerActive(ctx context.Context) (bool, error) {
 	return n.dr.SequencerActive(ctx)
 }
 
+func (n *adminAPI) SetLogLevel(ctx context.Context, lvlStr string) error {
```

Comment:
Probably out of scope for this PR, but this would be nice to also have in op-service so the batcher/proposer/challenger would automatically inherit this capability.

Reply:
Good suggestion, but we do not have a standard admin-RPC in the op-service yet. I'll open an issue for it, definitely nice to have.

Reply:
#7417